src/EFCore.PG/Metadata/NpgsqlRelationalAnnotations.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+using System.Linq;
+using JetBrains.Annotations;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Metadata
+{
+    /// <summary>
+    /// Provides relational-specific annotations specific to Npgsql.
+    /// </summary>
+    public class NpgsqlRelationalAnnotations : RelationalAnnotations, INpgsqlRelationalAnnotations
+    {
+        [NotNull]
+        public virtual IReadOnlyList<IPostgresExtension> PostgresExtensions
+            => PostgresExtension.GetPostgresExtensions(Metadata).ToList();
+
+        [NotNull]
+        public virtual IReadOnlyList<PostgresEnum> PostgresEnums
+            => PostgresEnum.GetPostgresEnums(Metadata).ToList();
+
+        [NotNull]
+        public virtual IReadOnlyList<PostgresRange> PostgresRanges
+            => PostgresRange.GetPostgresRanges(Metadata).ToList();
```

Comment:
It would be better to use `ToArray` since `ToList` produces the same array wrapped by a list. It's a nit, but could improve memory footprint.

Reply:
Good point, I'll make the change. One of these was already returning with `.ToList()`, but I don't think it's actually used for anything.